benchmarks/iperf/Makefile

```diff
@@ -1,8 +1,8 @@
 PLUGIN_NAME=		iperf
 PLUGIN_VERSION=		1.0
-PLUGIN_REVISION=	1
+PLUGIN_REVISION=	2
```

Comment:
we don't need the revision change as the dependency change gets updated automatically